tests/test_images.bzl

```diff
@@ -8,3 +8,11 @@ def pull_test_images():
         # Latest at "focal" tag
         digest = "sha256:9d6a8699fb5c9c39cf08a0871bd6219f0400981c570894cd8cbea30d3424a31f",
     )
+
+    oci_pull(
+        name = "ubuntu_jammy",
+        registry = "docker.io",
```

Comment:
I need to change this to some ghcr.io image, but I don't know how exactly we pushed `ubuntu_focal` in the first place. =)

Reply:
Trying public.ecr.aws/lts/ubuntu:jammy for the moment...

Reply:
Yeah, that didn't work. =)